packages/components/src/components/data-table/_data-table-action.scss

```
@@ -84,6 +84,14 @@
     pointer-events: all;
   }
 
+  .#{$prefix}--toolbar-search-container-expandable
+    .#{$prefix}--search--disabled
+    .#{$prefix}--search-magnifier {
+    background: $disabled-01;
```

```suggestion
    background-color: $disabled-01;
```

to avoid changing more properties than we need to